### DISCUSSION TITLE
用了controlnet后生成的图像直接都是线，怎么办呀，求助大佬们

### DISCUSSION THREAD

**Discussion by lizhaorui10** [Mikubill/sd-webui-controlnet#2993] [Q&A] [answered]
不同的介入时机终止时机都试过了，基本上都是这样的结果，辛苦大佬们解答
![屏幕截图 2024-07-11 023821](https://github.com/Mikubill/sd-webui-controlnet/assets/165266454/792daa42-8131-45fb-9554-84e0a710884e)

**Comment by huchenlei** [comment, marked as answer]
You are using wrong controlnet model. You should use openpose controlnet model instead of inpaint controlnet model.

**Reply by lizhaorui10** [reply]
感谢大佬
